apps/docs/src/app/core/component-docs/date-picker/examples/date-picker-format-example.component.ts

```diff
@@ -25,12 +25,12 @@ export const CUSTOM_FD_DATETIME_FORMATS: DateTimeFormats = {
     template: `
         <fd-date-picker [(ngModel)]="date" placeholder="dd-mm-yyyy"></fd-date-picker>
         <br />
-        <div>Selected Date: {{ date?.toDateString() }}</div>
+        <div>Selected Date: {{ date | dateFormatPipe }}</div>
```

Comment:
According to the other pipelines, and angular's naming convention, it's better to call the pipe `dateFormat`